src/constants/attributes.ts

```diff
@@ -23,6 +23,8 @@ export const KEY_EMB_W3C_TRACEPARENT = 'emb.w3c_traceparent';
 export const KEY_VIEW_NAME = 'view.name';
 export const KEY_EMB_NAVIGATION_SOURCE = 'emb.navigation_source';
 export const KEY_EMB_REFERRER_URL = 'emb.referrer_url';
+export const KEY_EMB_SUFRACE_NAME = 'app.surface.name';
+export const KEY_EMB_SUFRACE_ID = 'app.surface.id';
```

Comment:
```suggestion
export const KEY_EMB_SURFACE_NAME = 'app.surface.name';
export const KEY_EMB_SURFACE_ID = 'app.surface.id';
```

I'm a bit torn since this is the only place we refer to surface I also wonder if overall the SDK logic would be clearer with something like:

```suggestion
export const KEY_EMB_PAGE_PATH = 'app.surface.name';
export const KEY_EMB_PAGE_ID = 'app.surface.id';
```

either way I think could use a quick comment here explaining we're mapping the web specific page/route onto the more general client-side surface concept